src/Resources/views/CRUD/list__action_history.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+
+This file is part of SonataAdminBundle and based on @SonataAdmin\CRUD\list__action_show.html.twig.
+
+(c) Daniel Reiche <d.reiche@gmx.ch>
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+The 'history' routes and actions are already defined by SonataAdmin, they just lack a default list action template.
+We're using the 'link_action_history' message placeholder here, since this is already defined and well translated.
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
```

Comment:
Please use this header, like in the oder list_actions
```twig
{#

This file is part of the Sonata package.

(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>

For the full copyright and license information, please view the LICENSE
file that was distributed with this source code.

#}
```

Reply:
done